src/main/java/com/lootfood/db/repository/custom/PropertySetter.java

```diff
@@ -0,0 +1,24 @@
+package com.lootfood.db.repository.custom;
+
+import org.apache.commons.beanutils.PropertyUtils;
+import org.springframework.data.mongodb.core.query.Update;
+
+import java.beans.PropertyDescriptor;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Date;
+
+public class PropertySetter {
+    public static <T> void setFieldsToUpdate(T object, Update update) throws IllegalAccessException, NoSuchMethodException, InvocationTargetException {
```

Comment:
Catch all those exceptions `IllegalAccessException, NoSuchMethodException, InvocationTargetException ` and throw your custom RuntimeException. And remove throws from methods